global_helpers/panther_oss_helpers.py

```diff
@@ -291,12 +303,14 @@ def get_dictionary(key: str) -> dict:
         ) from exc
 
 
-def get_string_set(key: str) -> Set[str]:
+def get_string_set(key: str, force_ttl_check: bool = False) -> Set[str]:
     """Get a string set's current value (defaulting to empty set if key does not exit)."""
     response = kv_table().get_item(
         Key={"key": key},
         ProjectionExpression=_STRING_SET_COL,
```

Comment:
If i'm reading this correctly this will only pull back the stringset column and won't include the ttl

Reply:
Ah good catch, of course my live test was only on the dictionary one where that isn't the case 🤦 I was going to test the other two methods just for certainty's sake before merging just hadn't got to it yet!

Reply:
@darwayne do you think there is any downside to simply removing the `ProjectionExpression` parameter here, similar to how it works today in `get_dictionary`?

Reply:
There could be some edge case where a user has a ton of data in some other column that they aren't trying to retrieve. So tradeoffs are mostly latency based